scripts/lgshells/functions/fn_postInit.sqf

```diff
@@ -24,19 +24,20 @@ seekerCone = 120;  // in degree
 // unregister shell firing event handler when getting out of a vehicle
 player addEventHandler ["GetOutMan", {
   params ["_unit", "_role", "_vehicle", "_turret"];
-  _vehicle removeEventHandler ["Fired", lgshell_eventHandler_index];
```

Comment:
This is the lgshells bug I had to fix. Decided to move to vehicle namespace to keep it localized, unless there's an explicit need for global var here that I don't see? @NeutraleNull

Reply:
The one I mentioned in the discussion here https://github.com/gruppe-adler/Shoot_and_Scoot.Tanoa/pull/27#issuecomment-1637795166